CHANGELOG.md

```diff
@@ -2,6 +2,37 @@
 
 ## Unreleased
 
+### SDK
+
+* Remove unused dependencies, cleanup code after stabilizing Value
+  ([#6948](https://github.com/open-telemetry/opentelemetry-java/pull/6948))
+* Explicitly allow null into CompletableResultCode.failExceptionally()
+  ([#6963](https://github.com/open-telemetry/opentelemetry-java/pull/6963))
+
+#### Traces
+
+* Fix span setStatus
+  ([#6990](https://github.com/open-telemetry/opentelemetry-java/pull/6990))
+
+#### Logs
+
+* Add getters/accessors for readable fields in ReadWriteLogRecord.
+  ([#6924](https://github.com/open-telemetry/opentelemetry-java/pull/6924))
+
+#### Exporters
+
+* OTLP: Update to opentelemetry-proto 1.5
+  ([#6999](https://github.com/open-telemetry/opentelemetry-java/pull/6999))
+* Bugfix - OTLP: Ensure Serializer runtime exceptions are rethrown as IOException
+  ([#6969](https://github.com/open-telemetry/opentelemetry-java/pull/6969))
+* BREAKING - OTLP: Delete OTLP authenticator concept
```

Comment:
```suggestion
* BREAKING - OTLP: Delete experimental OTLP authenticator concept
```

Reply:
Good point. Also added a link to the new docs with code snippets on supported authentication options.